pkg/api/types.go

```diff
@@ -1346,6 +1346,7 @@ const (
 	ClusterProfileMetalPerfscaleCPT     ClusterProfile = "metal-perscale-cpt"
 	ClusterProfileMetalPerfscaleJetlag  ClusterProfile = "metal-perfscale-jetlag"
 	ClusterProfileMetalTelco5G          ClusterProfile = "metal-telco5g"
+	ClusterProfileMetalTelcoV10N		ClusterProfile = "metal-telcov10n"
```

Comment:
@kononovn, there seems to be a text alignment issue here...minor nitpick.

Reply:
+1 fixed.